src/prg-c-d.asm

```diff
@@ -50,234 +40,37 @@ sub_BANKC_801C:
-	.db $20, $90, $4C, $FD
-	.db $20, $B0, $4C, $FD
-	.db $20, $D1, $4B, $FD
-	.db $20, $F1, $0B, $FD, $FD, $FD, $28, $29, $29, $29, $29, $2A, $FD, $FD
-	.db $21, $11, $0B, $FD, $FD, $FD, $FD, $27, $FD, $FD, $27, $FD, $FD, $FD
-	.db $21, $31, $0B, $FD, $FD, $FD, $FD, $27, $FD, $FD, $27, $FD, $FD, $FD
-	.db $21, $51, $0B, $FD, $FD, $23, $24, $25, $22, $23, $24, $25, $25, $FD
-	.db $21, $71, $0B, $0B, $0C, $0D, $0E, $0F, $FD, $FD, $FD, $FD, $FD, $FD
-	.db $21, $95, $07, $1F, $1A, $30, $31, $32, $33, $1B
-	.db $21, $B5, $06, $53, $FC, $40, $41, $42, $43
-	.db $21, $D7, $03, $50, $51, $52
-	.db $21, $F6, $02, $20, $21
-	.db $23, $CB, $04, $44, $55, $A5, $65 ; Attribute table changes
-	.db $23, $D4, $03, $55, $5A, $56
-	.db $23, $DD, $02, $45, $15
-	.db $23, $E4, $01, $3F
-	.db $00
-
-; This is pointed to, but the very first byte
-; is the terminating 0, so nothing gets drawn.
-; This would have undone the attribute changes
-; done in the above PPU writing, but I guess
-; Nintendo realized they were never going to
-; use that part of the screen again
-MarioDream_DoNothing:
-	.db $00
-	.db $23, $CB, $44, $00
-	.db $23, $D4, $43, $00
-	.db $23, $DD, $42, $00
-	.db $00
-
-MarioDream_EraseBubble1:
-	.db $20, $8F, $4D, $FC
-	.db $20, $AF, $4D, $FC
-	.db $00
-
-MarioDream_EraseBubble2:
-	.db $20, $CF, $4D, $FC
-	.db $20, $EF, $4D, $FC
-	.db $00
-
-MarioDream_EraseBubble3:
-	.db $21, $10, $4C, $FC
-	.db $21, $30, $4C, $FC
-	.db $00
-
-MarioDream_EraseBubble4:
-	.db $21, $50, $4C, $FC
-	.db $21, $71, $4B, $FC
-	.db $00
-
-MarioDream_EraseBubble5:
-	.db $21, $95, $47, $FC
-	.db $21, $B5, $46, $FC
-	.db $21, $D7, $43, $FC
-	.db $21, $F6, $42, $FC
-	.db $00
-
-MarioDream_BubbleSprites:
-	.db $28, $00, $00, $A8
-	.db $28, $04, $01, $B0
-	.db $28, $08, $02, $C0
-	.db $28, $0C, $03, $B8
-
-byte_BANKC_8308:
-	.db $28, $02, $00, $A8
-	.db $28, $06, $01, $B0
-
-byte_BANKC_8310:
-	.db $28, $0A, $02, $C0
-	.db $28, $0E, $03, $B8
-
-MarioDream_SnoringFrameCounts:
-	.db $20
-	.db $0A
-	.db $0A
-	.db $0A
-	.db $0A
-	.db $0A
-	.db $0A
-	.db $20
-	.db $0A
-	.db $0A
-	.db $0A
-	.db $0A
-	.db $0A
-	.db $0A
-
-MarioDream_WakingFrameCounts:
-	.db $08
-	.db $08
-	.db $50
-	.db $40
-	.db $30
-	.db $10
-	.db $10
-
-MarioDream_SnoringFrames:
-	.db CHRBank_EndingBackground1
-	.db CHRBank_EndingBackground2
-	.db CHRBank_EndingBackground3
-	.db CHRBank_EndingBackground4
-	.db CHRBank_EndingBackground5
-	.db CHRBank_EndingBackground6
-	.db CHRBank_EndingBackground7
-	.db CHRBank_EndingBackground8
-	.db CHRBank_EndingBackground7
-	.db CHRBank_EndingBackground6
-	.db CHRBank_EndingBackground5
-	.db CHRBank_EndingBackground4
-	.db CHRBank_EndingBackground3
-	.db CHRBank_EndingBackground2
-
-MarioDream_WakingFrames:
-	.db $78
-	.db $74
-	.db $70
-	.db $7C
-	.db $70
-	.db $74
-	.db $78
-
+; ---------------------------------------------------------------------------
+;; PlayerXHi ;; <Dream_SceneCounter> = PlayerXHi ;; How many times this part of the scene plays
```

Comment:
you can add multiple labels in `ram.asm` and use those names in here instead, which might make the code read more clearly